src/components/json/worldData.js

```diff
@@ -524,6 +525,22 @@ const data = [
       "twitter": "https://twitter.com/_AnmolKaur_",
       
     }
+	{
+        "latitude": 28.5192,
+        "longitude": 77.3653,
+        "batch": 2020,
+        "photo": akshita2k,
+        "region": "Asia",
+        "skills": ["App Dev", "Web Dev", "ML", "AI", "Data Analysis", "Cloud"],
```

Comment:
Please select the skills present in the contributions doc only :)

Reply:
Made the required changes